otumat/watch.py

```diff
@@ -5,12 +5,17 @@
 
 
 class OnMyWatch:
-    def __init__(self, watch_file, watch_interval, watch_script, watch_args):
+    def __init__(self, watch_file, watch_interval, watch_script, watch_init, watch_args):
```

Comment:
I just realized we want the `initialize` stuff to be before the `script` stuff in all locations.

```suggestion
    def __init__(self, watch_file, watch_interval, watch_init, watch_script, watch_args):
```